server/protocols/jmap-rfc-8621/src/test/resources/sessionRoutesResult.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
I don't understand the added value of a file vs a multiline String

Reply:
some one prefer to look at the file because my test class is already too long :( how should I go?

Reply:
And what is the added value of multiline String vs file then @mbaechler ?